unstructured/partition/pdf_image/ocr.py

```diff
@@ -692,81 +409,18 @@ def supplement_layout_with_ocr_elements(
     return final_layout
 
 
-def get_elements_from_ocr_regions(
-    ocr_regions: List[TextRegion],
-    ocr_text: Optional[str] = None,
-    group_by_ocr_text: bool = False,
-) -> List[LayoutElement]:
-    """
-    Get layout elements from OCR regions
-    """
-
-    if group_by_ocr_text:
-        text_sections = ocr_text.split("\n\n")
-        grouped_regions = []
-        for text_section in text_sections:
-            regions = []
-            words = text_section.replace("\n", " ").split()
-            for ocr_region in ocr_regions:
-                if not words:
-                    break
-                if ocr_region.text in words:
-                    regions.append(ocr_region)
-                    words.remove(ocr_region.text)
-
-            if not regions:
-                continue
-
-            for r in regions:
-                ocr_regions.remove(r)
-
-            grouped_regions.append(regions)
-    else:
-        grouped_regions = cast(
-            List[List[TextRegion]],
-            partition_groups_from_regions(ocr_regions),
-        )
-
-    merged_regions = [merge_text_regions(group) for group in grouped_regions]
-    return [
-        LayoutElement(
-            text=r.text, source=r.source, type=ElementType.UNCATEGORIZED_TEXT, bbox=r.bbox
-        )
-        for r in merged_regions
-    ]
-
-
-def merge_text_regions(regions: List[TextRegion]) -> TextRegion:
-    """
-    Merge a list of TextRegion objects into a single TextRegion.
-
-    Parameters:
-    - group (List[TextRegion]): A list of TextRegion objects to be merged.
-
-    Returns:
-    - TextRegion: A single merged TextRegion object.
-    """
-
-    if not regions:
-        raise ValueError("The text regions to be merged must be provided.")
-
-    min_x1 = min([tr.bbox.x1 for tr in regions])
-    min_y1 = min([tr.bbox.y1 for tr in regions])
-    max_x2 = max([tr.bbox.x2 for tr in regions])
-    max_y2 = max([tr.bbox.y2 for tr in regions])
-
-    merged_text = " ".join([tr.text for tr in regions if tr.text])
-    sources = [tr.source for tr in regions]
-    source = sources[0] if all(s == sources[0] for s in sources) else None
-
-    return TextRegion.from_coords(min_x1, min_y1, max_x2, max_y2, merged_text, source)
-
-
 def get_ocr_agent() -> str:
-    ocr_agent = env_config.OCR_AGENT.lower()
-    if ocr_agent not in [OCR_AGENT_PADDLE, OCR_AGENT_TESSERACT]:
+    ocr_agent_module = env_config.OCR_AGENT
+    # deal with compatibility with origin way to set OCR
+    if ocr_agent_module.lower() == OCR_AGENT_TESSERACT_OLD:
+        ocr_agent_module = OCR_AGENT_TESSERACT
+    elif ocr_agent_module.lower() == OCR_AGENT_PADDLE_OLD:
+        ocr_agent_module = OCR_AGENT_PADDLE
```

Comment:
I'd suggest we also add a warning message here to let users know they are using a pattern that is outdated and would be deprecated later like 
```suggestion
        logger.warning("OCR agent name %s is outdated and will be deprecated in a future release; please use %s instead", ocr_agent_module, OCR_AGENT_TESSERACT)
        ocr_agent_module = OCR_AGENT_TESSERACT
    elif ocr_agent_module.lower() == OCR_AGENT_PADDLE_OLD:
        logger.warning("OCR agent name %s is outdated and will be deprecated in a future release; please use %s instead", ocr_agent_module, OCR_AGENT_PADDLE)
        ocr_agent_module = OCR_AGENT_PADDLE
```